core/src/main/java/com/exadel/aem/toolkit/core/optionprovider/services/impl/OptionProviderServiceImpl.java

```diff
@@ -184,4 +262,78 @@ private static void removeExcludedOptions(List<Option> options, String[] exclude
             .collect(Collectors.toList());
         options.removeAll(excludedOptions);
     }
+
+    /**
+     * Checks if the url valid
+     * @param urlString the url
+     * @return the result of check is url valid or not
+     */
+    private boolean isValidUrl(String urlString) {
+        try {
+            new URL(urlString);
+            return true;
+        } catch (MalformedURLException e) {
+            e.printStackTrace();
+        }
+        return false;
+    }
+
+    /**
+     * Extracts suffix from the url
+     * @param url the url
```

Comment:
(style) First, arg definitions must start with a capital letter (see all the pre-existed Javadoc). Second, "url - the url" is not at all good )) This is "doFoo() does foo". Third, if "url" is not the name of a Java variable but a common English word, it must be in caps. Even as I am typing this, Grammarly autocorrects "url" into "URL"